README.md

```diff
@@ -72,42 +72,42 @@ Different scenarios can also be run by making the following config updates from
+|            | [`rec_ippo.py`](mava/systems/ppo/anakin/rec_ippo.py)  | ✅         | ✅       | ✅     |         | [Link](https://arxiv.org/pdf/2011.09533) | [Link](mava/systems/ppo/README.md) |
+|            | [`rec_mappo.py`](mava/systems/ppo/anakin/rec_mappo.py) | ✅         | ✅       | ✅     |         | [Link](https://arxiv.org/pdf/2103.01955) | [Link](mava/systems/ppo/README.md) |
+| Q Learning | [`rec_iql.py`](mava/systems/q_learning/anakin/rec_iql.py)   |            | ✅       | ✅     |         | [Link](https://arxiv.org/pdf/1511.08779) | [Link](mava/systems/q_learning/README.md) |
+|            | [`rec_qmix.py`](mava/systems/q_learning/anakin/rec_qmix.py)  |            | ✅       | ✅     |         | [Link](https://arxiv.org/pdf/1803.11485) | [Link](mava/systems/q_learning/README.md) |
+| SAC        | [`ff_isac.py`](mava/systems/sac/anakin/ff_isac.py)   | ✅         |          | ✅     |         | [Link](https://arxiv.org/pdf/1801.01290) | [Link](mava/systems/sac/README.md) |
+|            | [`ff_masac.py`](mava/systems/sac/anakin/ff_masac.py)  | ✅         |          | ✅     |         |   /   | [Link](mava/systems/sac/README.md) |
+|            | [`ff_hasac.py`](mava/systems/sac/anakin/ff_hasac.py)  | ✅         |          | ✅     |         | [Link](https://arxiv.org/pdf/2306.10715) | [Link](mava/systems/sac/README.md) |
+| MAT        | [`mat.py`](mava/systems/mat/anakin/mat.py)       | ✅         | ✅       | ✅     |         | [Link](https://arxiv.org/pdf/2205.14953) | [Link](mava/systems/mat/README.md) |
+| Sable      | [`ff_sable.py`](mava/systems/sable/anakin/ff_sable.py)  | ✅         | ✅       | ✅     |         | [Link](https://arxiv.org/pdf/2410.01706) | [Link](mava/systems/sable/README.md) |
+|            | [`rec_sable.py`](mava/systems/sable/anakin/rec_sable.py) | ✅         | ✅       | ✅     |         | [Link](https://arxiv.org/pdf/2410.01706) | [Link](mava/systems/sable/README.md) |
 <h2>Environments</h2>
 
-These are the environments which Mava supports _out of the box_, to add a new environment, please use the existing wrapper implementations as an example. The framework indicates compatible agorithms, where Anakin algorithms work with JAX environments and Sebulba algorithms work with Numpy environments.
+These are the environments which Mava supports _out of the box_, to add a new environment, please use the [existing wrapper implementations](mava/wrappers/) as an example. We also indicate whether the environment is implemented in JAX or not. JAX-based environments can be used with algorithms that follow the Anakin distribution architecture, while non-JAX environments can be used with algorithms following the Sebulba architecture.
 
 
-| Environment                     | Action space        | JAX | Numpy |
+| Environment                     | Action space        | JAX | Non-JAX |
```

Comment:
Can we please add the original source repo for all the environments and their paper if there is one.

Reply:
Done